ui-admin/src/study/families/FamilyStructureTable.tsx

```diff
@@ -72,21 +71,10 @@ export const FamilyStructureTable = (
       }
     },
     {
-      id: 'shortcode',
-      header: 'Shortcode',
-      accessorKey: 'member.shortcode',
+      id: 'enrollee',
+      header: 'Enrollee',
       cell: ({ row }) => {
-        return <Link to={`${studyEnvContext.currentEnvPath}/participants/${row.original.member.shortcode}`}>
-          {row.original.member.shortcode}
-        </Link>
```

Comment:
the family structure table is.... wonky at best. I'll come by in a followup PR to make it more robust/able to handle more complex relationship structures.